specificatie/openapi.yaml

```diff
@@ -288,11 +315,8 @@ components:
           type: string
           title: WOZ-objectnummer
           description: Unieke identificatie van het WOZ-object
-        kadastraalOnroerendeZaakIdentificaties:
```

Comment:
Even als dubbelcheck, de 'kadastraalOnroerendeZaakIdentificaties' property komt te vervallen?

Reply:
Zoals het nu is gespecificeerd wel. Dat is nu de identificatie in de property KadastraalOnroerendeZaken.